mixer/docker/Dockerfile.mixer

```diff
@@ -1,22 +1,32 @@
 # BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
 ARG BASE_DISTRIBUTION=default
 
+# The following section is used as base image to add users
+# hadolint ignore=DL3006
+FROM istionightly/base_debug as base_debug
+COPY mixs /usr/local/bin/
+# 1001 is chosen as the UID to be added to the istio group as this is the UID that mixs in the docker container runs as.
+RUN addgroup --system --gid 1001 istio && \
```

Comment:
what is the UID of distroless `nonroot`? is it ok if its not the same 1001?

Reply:
the UID is 1002 for `nonroot`. I cannot imagine what problems we may have.